teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -9,7 +9,8 @@
     "Authors:\n",
```

Comment:
<p>The <code>IOError</code>  exceptions are usually applied for input/output errors that involve <em>file</em> reading, writing or access. In this case, we could argue that the value <code>aspirin</code>  for name is invalid, and hence we should raise <code>ValueError</code>.  Same for similar cells!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>done.</p>